src/util/parser.ts

```diff
@@ -0,0 +1,19 @@
+export const tryParseObject = (data: string | Buffer): Record<string, any> => {
```

Comment:
`Record<string, any>` is used for "anything json" here?

Reply:
Hm, yeah I thought we'd need to type the responses but actually we can replace that with `Message` :D